client/src/ui/organisms/footer/index.tsx

```diff
@@ -22,6 +22,16 @@ export function Footer() {
           </a>
           <p>Your blueprint for a better internet.</p>
           <ul className="social-icons">
+            <li>
+              <a
+                href="https://bsky.app/profile/mozdevnet.bsky.social"
```

Comment:
We've verified our domain, so the link should be updated here: https://bsky.app/profile/developer.mozilla.org

Reply:
That's great! 👍🏻 I just updated the link.